### PR TITLE
Implement warning for deprecated test networks, kovan, ropsten and rinkeby

### PR DESCRIPTION
**Pull request by tommasini**
**Description**
Kovan, Ropsten and Rinkeby are 3 test networks that will be deprecated soon with the merge of ethereum.

**Proposed Solution**
Created a warning with the information when the user change for one of this three networks.

**Screenshots/Recordings**
https://recordit.co/WeoxoIWOoo

**Issue**

Progresses #4697 

**Checklist**

* [x] There is a related GitHub issue
* [x] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Comment by bschorchit** [comment]
This looks great! Thank you, @tommasini

File: app/components/Nav/App/index.js [hunk 1]
```diff
@@ -359,6 +383,50 @@ const App = ({ userLoggedIn }) => {
...
+    Linking.openURL(
+      'https://blog.ethereum.org/2022/06/21/testnet-deprecation/',
+    );
+  };
+
+  const renderDeprecatedNetworkAlert = (network) => {
+    const { type } = network.provider;
+    if (type === ROPSTEN || type === RINKEBY || type === KOVAN) {
+      return (
+        <View style={styles.alertContainer}>
```
Comment (sethkfman) [inline app/components/Nav/App/index.js:396]: What do you think about moving this out to a separate component?

File: app/components/Nav/App/index.js [hunk 2]
```diff
@@ -359,6 +383,50 @@ const App = ({ userLoggedIn }) => {
     </Stack.Navigator>
   );
 
+  const openDeprecatedNetworksArticle = () => {
+    Linking.openURL(
+      'https://blog.ethereum.org/2022/06/21/testnet-deprecation/',
```
Comment (sethkfman) [inline app/components/Nav/App/index.js:388]: Suggest adding this to the constants/url.js

**Review by sethkfman — APPROVED** [review]
LGTM

**Review by cortisiko — CHANGES_REQUESTED** [review]
@tommasini found a couple 🐛 

In dark mode: there is a dark artifact in the warning alert https://www.screencast.com/t/42EScJP1ofhd 

In light mode: I cannot read the text in the warning alert:
https://www.screencast.com/t/VuL2JAB0A


Usability issue:
If I create a new wallet and I switch to a deprecated test net I see two warning alerts: The protect your wallet and network deprecation. Which one takes precedence? https://www.screencast.com/t/hbGAhZHyRS

Furthermore, If I were to open the burger menu I am unable to see the menu options to scroll through: http://recordit.co/ot71GMZndf

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮

**Comment by omnat** [comment]
Is this shown just once per user or everytime they switch to one of the 3 networks @tommasini ?

**Comment by tommasini** [comment]
The behaviour is:
 * Everytime they switch the network to one of the three the warning will prompt
 * If they close the alert it will only appear when they kill the app and re open it

**Pull request merged**